lib/diagnostic.js

```diff
@@ -7,13 +7,21 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function (/* */) {
-  /* your response here */
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    normalizedWords[i] = words[i].toUpperCase()
+  }
+  return normalizedWords
```

Comment:
Looks good here, although you simply forgot to remove punctuation. A simple way to do this is through regex.